src/main/java/ch/uzh/ifi/hase/soprafs24/service/GameEngineService.java

```diff
@@ -213,7 +213,9 @@ public void turnValidation(Long gameId, Long userId) throws IOException, Interru
 
         if (currentGame.isRepeatTurn()) {
             nextPlayer = terminatingUser;
-            //drawCardMoveTermination(gameId, userId, false);
```

Comment:
i think this is the reason why it doesn't work anymore

Reply:
That is the reason why we got 3 cards instead of two when a single attack is played.